frontend/ReleaseDashboard/TestDashboard.svelte

```diff
@@ -221,6 +223,24 @@
         }
     };
 
+    const shouldFilterByUser = function (user, test, group = undefined, tests = undefined, type = "test") {
```

Comment:
I propose to call it `shouldFilterOutByUser`

Reply:
Okay